ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -1565,16 +1619,26 @@ def test__execute_cleaning_foreign_drives(self,
             'is_reboot_required': constants.RebootRequired.false,
             'is_commit_required': False
         }
+        mock_commit_config.side_effect = ['42']
+        mock_change_physical_disk_state.return_value = {
+            'is_reboot_required': True,
+            'conversion_results': {
+            'RAID.Integrated.1-1': {'is_reboot_required': 'optional',
+                                    'commit_required': False,
```

Comment:
commit_required is no longer returned by python-dracclient, so we should delete the line above.

Reply:
Done